uint8 idx
float32 score
float32 x_percent
float32 y_percent
